### PR TITLE
Add Rod to the list

### PR DESCRIPTION
**Pull request by ysmood**
Use ABC order to insert the new item.

### CODE REVIEW

**Comment by ysmood** [comment]
@googlebot I signed it!

**Comment by ysmood** [comment]
Do I have to fix the lint? I don't think it's related to this PR.

**Comment by ysmood** [comment]
@paulirish Do I have to fix the lint? I don't think it's related to this PR.

**Comment by ysmood** [comment]
Hi @paulirish, what should I do to let this PR be merged?

File: readme.md [hunk 1]
```diff
@@ -71,6 +71,7 @@
   - [headless-devtools](https://github.com/cowchimp/headless-devtools) - Puppeteer plugin to get CSS Coverage or JS Heap snapshot.
   - [PuPHPeteer](https://github.com/rialto-php/puphpeteer) - PHP-bridge to control Puppeteer using PHP.
 - [Playwright](https://github.com/microsoft/playwright) - Node.js library to automate Chromium, Firefox and WebKit with a single API.
+- [Rod](https://github.com/go-rod/rod) - A Devtools driver for web automation and scraping.
```
Comment (paulirish) [inline readme.md:74]:
```suggestion
```

lets remove this. we don't need it in two places

**Review by paulirish — CHANGES_REQUESTED** [review]

**Pull request merged**